### BIG differences, this is diff of wrong files.
### CLUS "Type Specifier EQL"
### and CLHS topic selection http://clhs.lisp.se/Body/a_eql.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:eql.txt	2017-04-27 21:40:00.000000000 +0100
+++ saved-pages-clus/cl:types:eql.txt	2017-04-27 21:40:00.000000000 +0100
@@ -1,4 +1,31 @@
-EQL
-Please select which reference to EQL you intended
+Type Specifier EQL

-FunctionType Specifier
\ No newline at end of file
+Compound Type Specifier Kind
+Combining.
+Compound Type Specifier Syntax
+eql object
+Compound Type Specifier Arguments
+object - an object.
+Compound Type Specifier Description
+Represents the type of all x for which (eql object x) is true.
+The argument object is required. The object can be *, but if so it denotes itself (the symbol *) and does not represent an unspecified value. The symbol eql is not valid as an atomic type specifier.
+\
\ No newline at end of file
